### PR TITLE
633 - fix backslack comma behaviour

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Signed-off-by: Emily Jiang <emijiang6@googlemail.com>

### CODE REVIEW

**Comment by Emily-Jiang** [comment]
@radcortez @jbee please review

File: tck/src/main/java/org/eclipse/microprofile/config/tck/emptyvalue/EmptyValuesTestProgrammaticLookup.java [hunk 1]
```diff
@@ -120,6 +122,12 @@ public void testBackslashCommaStringGetValue() {
         Assert.assertEquals(value, "\\,");
     }
 
+    @Test
+    public void testBackslashCommaStringGetOptionalValueAsArrayorList() {
```
Comment (jbee) [inline tck/src/main/java/org/eclipse/microprofile/config/tck/emptyvalue/EmptyValuesTestProgrammaticLookup.java:126]:
```suggestion
    public void testBackslashCommaStringGetOptionalValueAsArrayOrList() {
```

**Review by jbee — APPROVED** [review]
Just a typo.

Shouldn't there also be an entry in the changelog or was this added after last release?

**Review by radcortez — APPROVED** [review]

**Comment by Emily-Jiang** [comment]
> Just a typo.
> 
> Shouldn't there also be an entry in the changelog or was this added after last release?

There was an entry on the release note.

**Comment by radcortez** [comment]
I think I've messed up when I tried this PR and didn't notice that we need to remove this piece:

https://github.com/eclipse/microprofile-config/blob/8d3ab2c6c41e74108754e7a6d713cb414e3aa6f0/tck/src/main/java/org/eclipse/microprofile/config/tck/emptyvalue/EmptyValuesTestProgrammaticLookup.java#L134

This is testing that `getOptionalValue` and `getOptionalValues` are not present if the value is a `\,` but with the updated rules, they now return `,`.

**Pull request merged**